api/app/signals/apps/signals/models.py

```diff
@@ -98,32 +98,20 @@ class Signal(CreatedUpdatedModel):
     def image(self):
         """ Field for backwards compatibility. The attachment table replaces the old 'image'
         property """
-        attachment = self._image_attachment()
-
+        attachment = self.attachments.filter(is_image=True).first()
         return attachment.file if attachment else ""
 
     @property
     def image_crop(self):
-        attachment = self._image_attachment()
-
-        if attachment:
-            return attachment.image_crop
-
-        return ""
-
-    @property
-    def attachments(self):
-        return Attachment.actions.get_attachments(self)
-
-    def _image_attachment(self):
-        return Attachment.actions.get_images(self).first()
+        attachment = self.attachments.filter(is_image=True).first()
+        return attachment.image_crop if self.image else ''
 
     class Meta:
         permissions = (
             ('sia_read', 'Can read from SIA'),
             ('sia_write', 'Can write to SIA'),
         )
-        ordering = ('created_at',)
+        ordering = ('created_at', )
```

Comment:
We should pick a style, I have seen these spaces added or removed recently --- this creates noise in PRs.

Reply:
Ok so let's pick 1. I don't mind, I think I just need some time to adjust using it